src/redux/index.js

```diff
@@ -52,23 +52,30 @@ function reducer(state = initialState, action) {
                ...state.todos.filter(obj => obj !== action.payload)
             ]
          }
-         // Still working on this
-      // case "COMPLETE":
-      //    let arr = [...state.todos]
-      //    let newArr = arr.find(obj => obj === action.payload).complete = true
-      //    console.log(newArr, "NEW ARR")
-      //    return {
-      //       ...state,
-      //       todos: [
-      //          ...state.todos,
-      //          state.todos.find(obj => obj === action.payload).complete = true,
-      //       ]
-      //    }
+      case "COMPLETE":
```

Comment:
Great job! It works, next step down the line will be to optimize and make better, but for now, this is great!